src/components/Picklist/index.js

```diff
@@ -257,7 +257,6 @@ class Picklist extends Component {
                         aria-describedby={errorMessageId}
                         autoComplete="off"
                         ref={this.triggerRef}
-                        aria-activedescendant={this.getAriaActivedescendant()}
```

Comment:
getAriaActivedescendant method should not be used in any other place, if this is true then remove it also